packages/opentelemetry-core/src/trace/sampler/ProbabilitySampler.ts

```diff
@@ -28,7 +28,9 @@ export class ProbabilitySampler implements Sampler {
   }
 
   shouldSample(parentContext?: SpanContext) {
-    // Respect the parent sampling decision if there is one
+    // Respect the parent sampling decision if there is one.
+    // TODO(legendecas): add an option to ignore parent regarding to spec:
```

Comment:
Can you add an issue number to this TODO?

Reply:
Opened a new issue to track this one https://github.com/open-telemetry/opentelemetry-js/issues/1284